src/vcpkg/commands.integrate.cpp

```diff
@@ -383,8 +384,9 @@ CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=%s"
         const int exit_code = cmd_execute_and_capture_output(cmd_line, get_clean_environment()).exit_code;
 
         const path nuget_package = buildsystems_dir / Strings::format("%s.%s.nupkg", nuget_id, nupkg_version);
-        Checks::check_exit(
-            VCPKG_LINE_INFO, exit_code == 0 && fs.exists(nuget_package), "Error: NuGet package creation failed");
+        Checks::check_exit(VCPKG_LINE_INFO,
+                           exit_code == 0 && fs.exists(nuget_package, IgnoreErrors{}),
+                           "Error: NuGet package creation failed");
```

Comment:
```suggestion
                           "Error: NuGet package creation failed with exit code %d", exit_code);
```

Just while you're here. It'd probably be better to actually capture the whole output and print it on error, but eh, this is a low-traffic code path.